spec/menu_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'menu'
+
+describe Menu do
+  it "is a hash" do
+    expect(subject.display).to be_a Hash
+  end
```

Comment:
Don't test for explicit types. If you need to change this type later on, this test will fail.

Better is:

```ruby
example_object = # construct example object
expect(test).to eq example_object
```